common/crypto/sig.h

```diff
@@ -15,6 +15,11 @@
 
 #pragma once
 #include <openssl/obj_mac.h>
+
+#ifndef PDO_USE_ECDSA_CURVE
+#define PDO_USE_ECDSA_CURVE NID_secp256k1
+#endif
+
 namespace pdo
```

Comment:
I get the compile time option... but did you forget to use it in the constants below?

Reply:
ouch - I missed to commit that :) Updated now